### DISCUSSION TITLE
Good NoSQL DB client suggestions?

### DISCUSSION THREAD

**Discussion by 8BitJonny** [Automattic/mongoose#11842] [Q&A] [answered]
I mean MongoDB compass is okay-ish but I feel like I don't know any client that has a good feature set, has good UI/UX and maybe even keyboard navigation as a plus.

Or what DB clients do you like?

**Comment by Uzlopak** [comment, marked as answer]
I recommend Robo 3T and Studio 3T.

**Reply by vkarpov15** [reply]
I usually use the MongoDB Atlas UI for prod, and the plain MongoDB shell for local. Haven't really found a client that I like.
